diff -U 3 -H -d -r -N -- dev-ezx-0.2.0-orig/include/ezx/ezxutilgraph.h dev-ezx-0.2.0/include/ezx/ezxutilgraph.h
--- dev-ezx-0.2.0-orig/include/ezx/ezxutilgraph.h	2005-11-22 06:36:44.000000000 +0700
+++ dev-ezx-0.2.0/include/ezx/ezxutilgraph.h	2006-07-31 01:20:51.000000000 +0700
@@ -33,6 +33,8 @@
 #include <qpixmap.h>
 
 class UTIL_Graph {
+// add public: line by Ketut P. Kumajaya <kumajaya@bluebottle.com>
+public:
 	static void drawUnderLine(QPainter*, QColorGroup const&,
 				  QPoint const&, QPoint const&);
 	static void drawOutlineText(QPainter*, QColor, QColor, int, int,
diff -U 3 -H -d -r -N -- dev-ezx-0.2.0-orig/include/ezx/ezxutilpushbutton.h dev-ezx-0.2.0/include/ezx/ezxutilpushbutton.h
--- dev-ezx-0.2.0-orig/include/ezx/ezxutilpushbutton.h	2005-11-22 06:37:22.000000000 +0700
+++ dev-ezx-0.2.0/include/ezx/ezxutilpushbutton.h	2006-07-31 01:25:24.000000000 +0700
@@ -83,7 +83,8 @@
 	UTIL_PushButton(QString const&, QWidget*, int, int, int);
 	UTIL_PushButton(QString const&, QString const&, QWidget*,
 			int, int, int);
-	virtual ~UTIL_PushButton();
+	// destructor not exist, removed by Ketut P. Kumajaya <kumajaya@bluebottle.com>
+	// virtual ~UTIL_PushButton();
 
 	QPixmap drawActive() const;
 	QPixmap drawNormal() const;
diff -U 3 -H -d -r -N -- dev-ezx-0.2.0-orig/include/ezx/zapplication.h dev-ezx-0.2.0/include/ezx/zapplication.h
--- dev-ezx-0.2.0-orig/include/ezx/zapplication.h	2005-11-22 06:37:37.000000000 +0700
+++ dev-ezx-0.2.0/include/ezx/zapplication.h	2006-08-01 00:48:08.000000000 +0700
@@ -163,7 +163,8 @@
 	/* Reimplemented virtual methods */
 	virtual ~ZApplication();
 	virtual bool eventFilter(QObject*, QEvent*);
-	virtual bool vieventFilter(QObject*, QEvent*);
+	// procedure not exist, removed by Ketut P. Kumajaya <kumajaya@bluebottle.com>
+	// virtual bool vieventFilter(QObject*, QEvent*);
 	virtual void timerEvent(QTimerEvent*);
 	virtual void setMainWidget(QWidget*);
 	virtual bool qwsEventFilter(QWSEvent*);
@@ -216,8 +217,10 @@
 	void resetPressHoldStatus(void);
 
 public:
-	int getMouseMode(QWidget*);
-	bool setMouseMode(QWidget*, int);
+	// changed int to static int by Ketut P. Kumajaya <kumajaya@bluebottle.com>
+	static int getMouseMode(QWidget*);
+	// changed bool to static void by Ketut P. Kumajaya <kumajaya@bluebottle.com>
+	static void setMouseMode(QWidget*, int);
 
 	int applyStyle(void);
 	bool getKBState(void);
diff -U 3 -H -d -r -N -- dev-ezx-0.2.0-orig/include/ezx/zpushbutton.h dev-ezx-0.2.0/include/ezx/zpushbutton.h
--- dev-ezx-0.2.0-orig/include/ezx/zpushbutton.h	2005-11-22 06:38:48.000000000 +0700
+++ dev-ezx-0.2.0/include/ezx/zpushbutton.h	2006-07-31 01:27:05.000000000 +0700
@@ -78,7 +78,8 @@
 	Q_OBJECT
 public:
 	/* Reimplemented virtual methods */
-	virtual ~ZPushButton();
+	// destructor not exist, removed by Ketut P. Kumajaya <kumajaya@bluebottle.com>
+	// virtual ~ZPushButton();
 	virtual void setMinimumSize(int, int);
 	virtual void setPalette(QPalette const&);
 
